doc/user_guide/messages/message_control.rst

```diff
@@ -193,6 +193,12 @@ Here's an example with all these rules in a single place:
             # error
             print(self.blop)
 
+Known limitations
```

Comment:
Let's not add this.

I am working on getting this fixed by changes in `astroid` and this is something that is easily missed when we finally get to it.

Reply:
Right, it's not more special than other open bugs.

Reply:
I agree I just moved it from the FAQ without thinking too much about it.